Source/AwsCommonRuntimeKit/mqtt/Mqtt5Client.swift

```diff
@@ -21,6 +33,29 @@ public class Mqtt5Client {
         aws_mqtt5_client_release(rawValue)
     }
 
+    public func start() throws {
+        let errorCode = aws_mqtt5_client_start(rawValue)
+        if errorCode != 0 {
+            throw CommonRunTimeError.crtError(CRTError(code: errorCode))
+        }
+    }
+
+    public func stop(disconnectPacket: DisconnectPacket? = nil) throws {
```

Comment:
Think about it, as we only have one parameter, we could use '_' so the user does not need explicit call out the parameter name here. 
```suggestion
    public func stop(_ disconnectPacket: DisconnectPacket? = nil) throws {
```